docs/how-to/import-prisma-schema.md

```diff
@@ -0,0 +1,120 @@
+---
+id: import-prisma-schema
+title: Import Prisma Schema
+sidebar_label: Import Prisma Schema
+slug: /how-to/import-prisma-schema
+pagination_next: getting-started/how-prisma-schema-is-converted-into-entities
+---
+
+# Import Prisma Schema
+
+Amplication's **Import Prisma Schema** feature lets you use your _existing_ databases for a head start on development.
+
+This way you don't have to manually create entities, fields, and relationships from scratch saving you a lot of time.
+
+Just upload your `schema.prisma` file.
+Amplication will then search for models and [convert them into Amplication's internal structure](/getting-started/how-prisma-schema-is-converted-into-entities).
+
+This guide walks you through how to generate a custom `schema.prisma` file from your existing database and upload it to Amplication.
+
+## How to create and upload your `schema.prisma` file
+
+First, you will run the Prisma introspection process on your existing database.
+The introspection process will generate a schema file representing your database's tables and columns.
+
+Next, you'll upload the generated file into Amplication, ensuring it's free of errors before doing so.
+After upload, you'll see an upload log of everything that happens to [convert the schema into Amplication's internal structure](/getting-started/how-prisma-schema-is-converted-into-entities).
+If any warnings appear in the upload log you can then choose to modify your `schema.prisma` file and re-upload it or continue as is.
```

Comment:
This isn't very clear... I was looking for details... we can skip the redundancy and just start with the steps or make sure the first paragraph does not sound like instructions.